### DISCUSSION TITLE
OpenTelemetry + Signoz APM

### DISCUSSION THREAD

**Discussion by lvanoverberghe** [chatwoot/chatwoot#8170] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

I would like to propose the addition of integrating OpenTelemetry and Signoz for APM in Chatwoot.

### Describe the solution you'd like

Same solution as now with Datadog/New Relic with environment variables

**Comment by pranavrajs** [comment]
Moving to discussions. I'm not sure if Signoz has Ruby binding, will ask the maintainers and see if it is feasible.
